payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/resources/StaticAddressResourceRepository.kt

```diff
@@ -0,0 +1,22 @@
+package com.stripe.android.ui.core.forms.resources
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.ui.core.address.AddressRepository
+
+/**
+ * [StaticAddressResourceRepository] that receives address resources pre-loaded.
+ */
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class StaticAddressResourceRepository(
```

Comment:
This and `StaticLpmResourceRepository` can be a generic `StaticRepository` that take the repository as parameter. Not a big deal to keep it as is too.